### DISCUSSION TITLE
Missing 5kb TSS files for human in cistargets resources database

### DISCUSSION THREAD

**Discussion by pcm32** [aertslab/SCENIC#403] [* General SCENIC questions] [answered]
Hi there!

Thanks for this great tool! We noticed that in: 

https://resources.aertslab.org/cistarget/databases/homo_sapiens/hg38/refseq_r80/mc_v10_clust/gene_based/ 

The 5kb upstream files are mentioned in the readme shown before the file listing, but they are not actually available for download among the listed files. Would it be possible for those to be uploaded there if someone in your lab has them? At least the TSS+/-5kb would be great.

Thanks!

![image](https://github.com/aertslab/SCENIC/assets/368478/ca26d553-a6cf-4530-98a5-cc1c00f179ba)

**Comment by ghuls** [comment, marked as answer]
TSS+/-5kb databases were never created for the most recent databases, only:
  - TSS+/-10kb
  - 500bp up / 100bp down TSS

Readme is corrected now.
